doc/langref.html.in

```diff
@@ -10901,9 +10901,15 @@ export fn add(a: i32, b: i32) i32 {
 }
       {#code_end#}
       <p>To make a static library:</p>
-      {#shell_samp#}$ zig build-lib mathtest.zig{#end_shell_samp#}
+      {#shell_samp#}$ zig build-lib -femit-h mathtest.zig{#end_shell_samp#}
       <p>To make a shared library:</p>
-      {#shell_samp#}$ zig build-lib mathtest.zig -dynamic{#end_shell_samp#}
+      {#shell_samp#}$ zig build-lib -femit-h mathtest.zig -dynamic{#end_shell_samp#}
+      <p>
+      The <kbd>-femit-h</kbd> flag is necessary in order to generate the mathtest.h file.
+      The generated C header file includes the zig.h file that must be copied
+      to the project directory, as an example using <a href="https://stedolan.github.io/jq/">jq</a>:
+      </p>
+      {#shell_samp#}cp $(zig env | jq -r .lib_dir)/zig.h .{#end_shell_samp#}
```

Comment:
A command like this shouldn't be needed to use `-femit-h` and should be fixed before documenting `-femit-h`.

Reply:
The command is not needed to use `-femit-h`, but to copy the `zig.h` file to the current directory.  Without the `zig.h` file `zig build` fails on my system.

Thanks.

Reply:
Did you set the `CPATH` environment variable manually?

I also tried with the new official zig 0.10.0 package from Archlinux, but `zig.h` is not found (it is installed in `/usr/lib/zig/include/zig.h`.

Reply:
I didn't set anything, I get all the same errors as you do. What I meant is that you shouldn't need to copy anything or set some environment variable, `zig.h` should either not be referenced by the emitted header or it should be installed alongside it.

Reply:
Ok, I misunderstood what you wrote.  Thanks.
I will open an issue.